File: CS/DxBlazorApplication1/Components/MDI/MDITab.cs
namespace DxBlazorApplication1.Components.MDI
{
    public class MDITab {
        public string Text { get; set; }
        public int VisibleIndex { get; set; }
        public bool Visible { get; set; }
        public MDITab(string text, int visibleIndex, bool visible) {
            Text = text;
            VisibleIndex = visibleIndex;
            Visible = visible;
        }
    }
}

File: VB/DxBlazorApplication1/Components/MDI/MDITab.vb
Namespace DxBlazorApplication1.Components.MDI

    Public Class MDITab

        Public Property Text As String

        Public Property VisibleIndex As Integer

        Public Property Visible As Boolean

        Public Sub New(ByVal text As String, ByVal visibleIndex As Integer, ByVal visible As Boolean)
            Me.Text = text
            Me.VisibleIndex = visibleIndex
            Me.Visible = visible
        End Sub
    End Class
End Namespace
